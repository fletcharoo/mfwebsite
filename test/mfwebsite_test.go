package test

import (
	"io"
	"net/http"
	"testing"
)

func Test_HelloWorld(t *testing.T) {
	resp, err := http.Get("http://localhost:8080")

	if err != nil {
		t.Fatalf("failed to GET: %s", err)
	}

	body, err := io.ReadAll(resp.Body)

	if err != nil {
		t.Fatalf("failed to read body: %s", err)
	}

	if string(body) != "Hello, world!" {
		t.Fatalf("unexpected response: %s", err)
	}
}
