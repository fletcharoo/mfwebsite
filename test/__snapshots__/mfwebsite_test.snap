
[Test_index - 1]
<!DOCTYPE html>
<html>
<head>
  <title></title>
  <meta name="GENERATOR" content="github.com/gomarkdown/markdown markdown processor for Go">
  <meta charset="utf-8">
</head>
<body>

<p>This is the home page</p>

</body>
</html>

---

[Test_other - 1]
<!DOCTYPE html>
<html>
<head>
  <title></title>
  <meta name="GENERATOR" content="github.com/gomarkdown/markdown markdown processor for Go">
  <meta charset="utf-8">
</head>
<body>

<p>This is another page</p>

</body>
</html>

---

[Test_styleCSS - 1]
body {
    margin: 40px auto;
    max-width: 650px;
    line-height: 1.4;
    font-size: 18px;
    color: #444;
    background-color: #EEE;
    padding: 0 10px;
}

h1,h2,h3,h4,h5,h6 {
    line-height: 1.2;
}

---

[Test_subdir - 1]
<!DOCTYPE html>
<html>
<head>
  <title></title>
  <meta name="GENERATOR" content="github.com/gomarkdown/markdown markdown processor for Go">
  <meta charset="utf-8">
</head>
<body>

<p>This is in a subdirectory</p>

</body>
</html>

---
